SQL/TestingScratchpad.sql

```diff
@@ -1,35 +1,20 @@
+-- SPUT
+call WebCrawler.TEST_wcSp_GetExistingTree();
+
 -- Testing Selects 
 SELECT * FROM wc_TempEdges;
+SELECT * FROM wc_Edges;
 SELECT * FROM wc_Users;
 SELECT * FROM wc_Urls;
-SELECT * FROM wc_Edges;
-SELECT * FROM wc_UserSearches WHERE Id = 92;
-SELECT * FROM wc_SingleGraphs WHERE UserSearchId = 92;
-
--- Order of operations 
-SET @resLocal = -99;
-CALL wcSp_DoesCookieExist(11, @resLocal);
-SELECT @resLocal; SET @resLocal = -99;
-
-CALL wcSp_GetExistingTree(8);
-SELECT @resLocal; SET @resLocal = -99;
-
-CALL wcSp_ReturnSetOfDestUrlsForSrc("http://9");
-SELECT @resLocal; SET @resLocal = -99;
-
-CALL wcSp_InsertIntoTempEdges("http://15", "http://16", 11, @resLocal);
-SELECT @resLocal; SET @resLocal = -99;
-
-CALL wcSp_InsertNewTree("http://newCookie", "RogerPedacker", 11, "BFS", 22, @resLocal);
-SELECT @resLocal; SET @resLocal = -99;
-
-DELETE FROM wc_TempEdges;
-SET @url1 = CONCAT('http://', CONCAT(CAST(unix_timestamp() AS char), '1'));
```

Comment:
I mean that all these tests were removed. I was just wondering why, but it looks like you added a bunch of javascript tests instead.